### DISCUSSION TITLE
repository-container-header disappearing on clicking the browser back button

### DISCUSSION THREAD

**Discussion by Koboki** [community/community#10219] [Projects and Issues] [answered]
1- Open the "issues" tab for any repository (React for example https://github.com/facebook/react/issues).
2- Click the link that says "we've collected some good first issues for you."
3- Click on the browser's back button.
4- The "repository-container-header" element disappears (refreshing the page will make it show up again).

Tested on Chrome and Firefox on Windows 10.

https://user-images.githubusercontent.com/6704238/149765249-fb14d56b-e672-4c27-a2cc-397020221b0c.mp4

**Comment by anleac** [comment, marked as answer]
👋 thanks for reporting this - I can confirm a repro, we'll have a look into it on our side, hopefully it's an easy fix and can update shortly.
